### PR TITLE
ci: add relup version db

### PR DESCRIPTION
**Pull request by thalesmg**
Follow up to https://github.com/emqx/emqx/pull/8684#issuecomment-1219230701

### CODE REVIEW

File: scripts/relup-base-vsns.escript [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    io:format(user, "~s~n", [OtpVsn]),
+    halt(0);
+main(["insert-new-vsn", NewVsn0, BaseFromVsn0, OtpVsn0, VsnDB]) ->
+    {ok, [VsnMap]} = file:consult(VsnDB),
+    NewVsn = strip_pre_release(NewVsn0),
+    BaseFromVsn = strip_pre_release(BaseFromVsn0),
+    OtpVsn = list_to_binary(OtpVsn0),
+    case VsnMap of
+      #{NewVsn := _} ->
+          io:format(user, ?RED ++ "Version ~s already in DB!~n" ++ ?RESET, [NewVsn]),
```
Comment (ieQu1) [inline scripts/relup-base-vsns.escript:32]: Nit: maybe it's easier to create a function "print_warning" that encapsulates the escape codes?

Comment (thalesmg) [reply scripts/relup-base-vsns.escript]: :heavy_check_mark:

File: scripts/relup-base-vsns.escript [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                io:format(user, "  ~s~n", [Vsn])
+              end, Candidates),
+            print_warning(
+              "To insert this version automatically, run:~n"
+              "./scripts/relup-base-vsns insert-new-vsn NEW-VSN BASE-FROM-VSN NEW-OTP-VSN ~s~n"
+              "And commit the results.  Be sure to revise the changes.~n"
+              "Otherwise, edit the file manually~n",
+              [VsnDB]),
+            halt(1)
+    end.
```
Comment (ieQu1) [inline scripts/relup-base-vsns.escript:77]: It would be nice to have some help message that explains the purpose of the script and how to use it.

**Review by ieQu1 — APPROVED** [review]

File: scripts/relup-base-vsns.escript (hunk 2 above)
Comment (thalesmg) [reply scripts/relup-base-vsns.escript:77]: Indeed.  I'll follow up this PR with an usage / description message.

**Comment by thalesmg** [comment]
I'll open a new PR containing theses changes, an usage message and move the `.eterm` file to the `rel` dir.

**Comment by thalesmg** [comment]
Superseded by #8776

**Pull request closed without merging**